relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -2,6 +2,7 @@ use std::collections::BTreeMap;
 
 use relay_common::{DurationUnit, EventType, MetricUnit, SpanStatus, UnixTimestamp};
 use relay_dynamic_config::{AcceptTransactionNames, TaggingRule, TransactionMetricsConfig};
+use relay_filter::csp::SchemeDomainPort;
```

Comment:
You could probably also use `url::Url` to reduce cross-dependencies.

Reply:
TIL, thanks! I'll do that in a follow-up once we've rolled this out.